core/src/main/java/com/google/errorprone/bugpatterns/nullness/VoidMissingNullable.java

```diff
@@ -86,6 +87,11 @@ public class VoidMissingNullable extends BugChecker
   @Override
   public Description matchParameterizedType(
       ParameterizedTypeTree parameterizedTypeTree, VisitorState state) {
+    if (!hasExplicitSource(parameterizedTypeTree, state)) {
+      /* Implicit types cannot be annotated. */
+      return NO_MATCH;
+    }
```

Comment:
We could expand the comment [like here](https://github.com/google/error-prone/blob/43a6c45b20acb4395913eeb152b1d06e28c10022/core/src/main/java/com/google/errorprone/bugpatterns/nullness/VoidMissingNullable.java#L144-L152), but I'm not sure that we want to make a similarly strong statement about inference of local parameterized types.